src/plugins/presentation_panel/public/panel_component/panel_header/presentation_panel_hover_actions.tsx

```diff
@@ -515,12 +520,12 @@ export const PresentationPanelHoverActions = ({
         &:focus-within,
         &.embPanel__hoverActionsAnchor--lockHoverActions {
           .embPanel {
-            outline: ${viewMode === 'edit' ? DASHED_OUTLINE : SOLID_OUTLINE};
-            z-index: ${euiThemeVars.euiZLevel2};
+            outline: ${viewMode === 'edit' ? EDIT_MODE_OUTLINE : VIEW_MODE_OUTLINE};
+            z-index: ${euiTheme.levels.menu};
           }
           .embPanel__hoverActionsWrapper {
-            z-index: ${euiThemeVars.euiZLevel9};
-            top: -${euiThemeVars.euiSizeXL};
+            z-index: ${euiTheme.levels.modal};
```

Comment:
Question: Is this change expected? The value of `euiZLevel9` was `9000` and `euiTheme.levels.modal` is `8000`.
If it should still be `9000` then you can use `euiTheme.levels.toast`

📖 [Level docs](https://eui.elastic.co/#/theming/more-tokens#levels)

Reply:
I don't think it absolutely needs to be 9000. I may actually reduce it to a lower level bc it really shouldn't appear over modals, toasts, popovers, etc.